### ISSUE TITLE
Cannot set property 'pdfjsLib' of undefined

### ISSUE THREAD

**Issue by youngList** [wojtekmaj/react-pdf#808]
**Before you start - checklist**

 - [x] I followed instructions in documentation written for my React-PDF version
 - [x] I have checked if this bug is not already reported
 - [x] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
 - [x] If I have a problem with PDF rendering, I checked if my PDF renders properly in Mozilla Firefox

**Description**
I need to show PDF in chrome that version is 49.0.2623.112, so I compile pdfjs-dist with babel, I get this wrong message, "Cannot set property 'pdfjsLib' of undefined".
then I find that  there are "For usage with older browsers/environments, without support for modern features
such as e.g. `async`/`await`, `Promise`, and `ReadableStream`,
please refer to the `es5` folder." in readme.md in the pdfjs-dist, I want to konw how to do for old browsers with react-Pdf?

**Expected behavior**

I can use react-pdf in old browsers, tks

**Environment**

 - Browser (if applicable) [e.g. Chrome 57, Firefox 59]: Chrome 49
 - React-PDF version [e.g. 3.0.4]: 5.3.0
 - React version [e.g. 16.3.0]: 16.14.0
 - Webpack version (if applicable) [e.g. 4.16.2]: 5.34.0

**Comment by wojtekmaj** [comment]
Might get fixed by #794

**Comment by chinnick967** [comment]
I can affirm that our project is having this same issue. I had to create a loader to compile pdfjs-dist using Babel due to another issue, and doing that causes this error.

**Comment by wojtekmaj** [comment]
Please kindly test 5.4.0-beta.3.

**Comment by chinnick967** [comment]
So it doesn't look like that latest build resolved this issue.

When trying to build this in webpack 4, webpack requires an additional loader due to pdfjs-dist using optional chaining. I have to create a specific loader for handling the pdfjs-dist module to load it using babel, and then it builds correctly but I begin getting this error instead in the console and it breaks the project.

**Comment by wojtekmaj** [comment]
5.4.0-beta.3 should not contain optional chaining in legacy build of pdfjs-dist. 🤔

**Comment by jaredmeakin** [comment]
Just tested out v5.4.0 and getting the following error when trying to build with Webpack 4 and Node v10.24.1

```
ERROR in ./node_modules/pdfjs-dist/build/pdf.js 2164:21
Module parse failed: Unexpected token (2164:21)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     var _intentState;
| 
>     if (arguments[0]?.annotationStorage !== undefined) {
|       (0, _display_utils.deprecated)("render no longer accepts an `annotationStorage` option, " + "please use the `includeAnnotationStorage`-boolean instead.");
|       includeAnnotationStorage || (includeAnnotationStorage = !!arguments[0].annotationStorage);
```

**Comment by wojtekmaj** [comment]
Hmmm, you should never have had this file imported in the first place, do you have a stack where it was imported from by any chance? Or are you importing something from pdfjs-dist yourself?

**Comment by jaredmeakin** [comment]
You're right. Looks like we were pulling in `pdfjs-dist` in another file I wasn't aware of. Cleared that up, and now I'm just getting `Error: Can't resolve 'react-pdf'`. It is indeed installed and at 5.4.0 verified by running `npm list react-pdf`. Any thoughts on why it would fail to resolve?

```
ERROR in ./src/features/Forms/PDFviewer/components/DocumentViewer/index.jsx
Module not found: Error: Can't resolve 'react-pdf' in '/Users/jared.meakin/Code/nora-frontend/src/features/Forms/PDFviewer/components/DocumentViewer'
 @ ./src/features/Forms/PDFviewer/components/DocumentViewer/index.jsx 26:16-36
 @ ./src/components/_Common/FileUpload/index.jsx
 @ ./src/components/_Common/index.js
 @ ./src/entry/Main.jsx
 @ ./src/entry/Root.jsx
 @ ./src/index.jsx
 @ multi webpack-hot-middleware/client?path=/__webpack_hmr&timeout=20000 ./src/index.jsx
```

**Comment by chinnick967** [comment]
Yup, just want to verify that this issue continuing for us was caused by us importing pdfjs in another part of our application and after changing it to point to the legacy version we no longer had this error. This issue can probably be closed as resolved with the 5.4 release.
